cmake/modules/FindEPOLL.cmake

```diff
@@ -9,6 +9,11 @@
 # The hint can be given on the command line too:
 #   cmake -DEPOLL_PATH_HINT=/DATA/ERIC/EPOLL /path/to/source
 
+if (FREEBSD)
```

Comment:
Does something need to be added to detect kevent?  Or is kevent always available?

Reply:
According to man kevent(2):
`The kqueue() and kevent() system calls first appeared in FreeBSD 4.1.`
and AFAIK there is no way to disable kevent at compile-time nor runtime,
so it should always be available.

Reply:
Good enough.